setup.py

```diff
@@ -28,6 +28,7 @@
     "tests": tests_require,
     "kubernetes": ["kubernetes>=11.0.0,<12.0.0",],
     "yadage": ["adage==0.10.1", "yadage==0.20.1", "yadage-schemas==0.10.6",],
+    "snakemake": ["snakemake==6.5.2"],
```

Comment:
Do we need the exact version pin?  E.g. now there is `6.5.3` released, so 6.5.2 is already outdated. Perhaps we can have more relaxed boundaries? I aven't looked at the Snakemake ecosystem practices whether `major.minor.patchlevel` philosophy is well respected; if yes, then we could allow `6.5.*` and still be fine?  (allowing `6.*` would be most probably too much, since APIs would probably change there...)